### PR TITLE
WIP: fix: temp enable nft prices to show

### LINKED ISSUE #3422: buying NFT doesn't change it's price to 0
When testing gallery, I can still see NFT which had last event 'BUY' with price, such as:
https://beta.kodadot.xyz/bsx/gallery/2645998698-6
https://kodadot.xyz/bsx/gallery/2645998698-6
image

Upon clicking on NFT, I cannot buy it anymore (which is correct behavior).

image

Observing things in [snek playground](https://app.gc.subsquid.io/beta/snekk/003/graphql) shows the price there, same thing in rubick results in price 0 (unlisting nft after buy event)
image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3422 
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Review by petersopko — COMMENTED** [review]
I don't think this is the desired outcome,  you've made the NFT buyable again, but I'd expect the NFT to be taken off the 'market' once you buy it. This is the equivalent of buying and relisting automatically.

**Comment by vikiival** [comment]
> Do you have some idea to deal with it 👀 @vikiival

The `observePrice` function is correct. The NFT has price on the chain

Therefore I opened issue in the Basilisk repo:
- https://github.com/galacticcouncil/Basilisk-node/issues/521

The reason why you don't see the price from the indexer is that I forcefully set price to `undefined`

https://github.com/kodadot/snek/blob/f060f04575d2a3ed167c69884a29b1ad9957748e/src/mappings/index.ts#L220

However, my implementation is not yet correct :)


To sum it up.

I am closing this PR as the root of this issue is external
